conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/FrameworkException.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.runtime;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.logsafe.Arg;
+import com.palantir.logsafe.SafeLoggable;
+import io.undertow.util.StatusCodes;
+import java.util.List;
+
+/** Internal type to signal a conjure protocol-level failure with a specific response code. */
+final class FrameworkException extends RuntimeException implements SafeLoggable {
```

Comment:
This entire class exists just because conjure doesn't support Unprocessable entity / unsupported media type right?

https://github.com/palantir/conjure-java-runtime-api/blob/develop/errors/src/main/java/com/palantir/conjure/java/api/errors/ErrorType.java#L39-L48

Reply:
That is correct, because unprocessable entity is an rpc concern that we would not want to expose for services to throw.

Sorry I missed the notification on this last week!